Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -642,7 +656,7 @@ internal struct LongPathInfo
                     }
                 };
 
-            recheck:
+                recheck:
```

Comment:
this is what the formatter did... so i'll just leave it as is... otherwise each time you format the document you have to put it back, which is annoying.